### ISSUE TITLE
USB Devices stop working

### ISSUE THREAD

**Issue by RodriMora** [xxxzc/xps15-9570-macos#39]
Hi!

Everything works fine. But after some time (I cannot tell how much, sometime its 10 min, sometimes it's 30min) the USB devices stop working. 
i'm using a usb 3.0 hub with a network card to  run wired internet and also a Acer predator monitor with another hub integrated, both stop working, but the keyboard and trackpad work just fine. 
It doens't matter if I reconnect them, once they stop working they don't work anymore until a reboot.
Both work fine in windows.
Using latest version of catalina 10.15.4

**Comment by RodriMora** [comment]
Any Idea what could it be? What logs should I attatch?

**Comment by xxxzc** [comment]
I've tried many methods to solve this issue before, but nothing works :(.

**Comment by xxxzc** [comment]
Do you have USB 2.0 device connected to USB hub? In my experience, connecting only USB 3.0 devices(removable hard drive, monitor...) to USB hub will not cause this problem.

**Comment by RodriMora** [comment]
Yes I do, the hub is USB 3.0 but have connected both USB 2 and 3 to the hub. Will test with only 3.0

**Comment by xxxzc** [comment]
Temporary solution: sleep and wake(fn + insert)
